opentracing_instrumentation/client_hooks/_dbapi2.py

```diff
@@ -81,6 +81,48 @@ def empty_ctx_mgr():
     )
 
 
+class CursorWrapper(wrapt.ObjectProxy):
```

Comment:
why move this code?

Reply:
Hi @yurishkuro,

I believe that @condorcet moved it because `CursorWrapper` must be defined before it's used to set `cursor_cls` attribute of `ConnectionWrapper` class.

Reply:
Absolutely right, that's why I moved this code